README.md

```diff
@@ -1,13 +1,28 @@
 [![Build Status](https://travis-ci.com/bids-standard/bids-specification.svg?branch=master)](https://travis-ci.com/bids-standard/bids-specification)
 [![CircleCI](https://circleci.com/gh/bids-standard/bids-specification.svg?style=svg)](https://circleci.com/gh/bids-standard/bids-specification)
 
```

Comment:
adding the twitter badge in
```suggestion
[![@BIDSstandard](http://img.shields.io/twitter/follow/bidsstandard.svg?style=social)](https://twitter.com/BIDSstandard)
```

Reply:
appears https://github.com/bids-standard/bids-specification/pull/268 takes care of adding the badge. added this to ensure the version consistency

Reply:
I won't commit this suggestion, then, so I don't create a merge conflict ! Let me know if I'm misunderstanding.

Reply:
We can keep this hear in case it is needed. It depends on the order of merging since we have a number of PRs open with differing versions of text. I may be mistaken too.

Reply:
I imagine #268 will merge first since I have zero approving reviews (having successfully dismissed my previous two :sweat_smile: ). I'll just merge in master if something breaks when it's merged !

Reply:
Sounds good!

Reply:
> I'll just merge in master if something breaks when it's merged !

yep, that's the way to go :-) (or even better: `git rebase master` from your branch, because then we won't have a "merge commit" --> these are often making the history a bit dirty I find)

Reply:
one caveat though is this will cause the changelog to go out of sync (until a merge commit is seen then it will catch up). the merge commit message is the signal for the changelog generator to generate

Reply:
I went ahead and merged #268, if you want to merge/rebase, @emdupre. Though it's not obvious that it won't merge cleanly as is. GitHub isn't indicating a merge conflict, but I can't remember if it does that before all reviews are in.